docs/tutorial/timeseries.rst

```diff
@@ -4,74 +4,46 @@
-
-This is calling the `~sunpy.timeseries.TimeSeries` factory to create a time series from a GOES XRS FITS file.
-
-The variable ``my_timeseries`` is a `~sunpy.timeseries.GenericTimeSeries` object.
-To create one from a local GOES/XRS FITS file try the following:
-
-.. code-block:: python
-
-    >>> my_timeseries = ts.TimeSeries('/mydirectory/myts.fits', source='XRS')   # doctest: +SKIP
-
-**sunpy** will attempt to detect automatically the instrument source for most FITS files.
-However time series data are stored in a variety of file types (FITS, txt, csv, CDF), and so it is not always possible to detect the source.
-**sunpy** ships with a number of known instrumental sources, and can also load CDF files that conform to the `Space Physics Guidelines for CDF <https://spdf.gsfc.nasa.gov/sp_use_of_cdf.html>`__.
-If you would like **sunpy** to include another instrumental source see the `Newcomers' Guide <https://docs.sunpy.org/en/latest/dev_guide/contents/newcomers.html>`__.
-
-The `~sunpy.timeseries.TimeSeries` factory has the ability to create a list of TimeSeries objects using a list of filepaths, a folder or a glob, for example:
-
-.. code-block:: python
-
-    >>> my_ts_list = ts.TimeSeries('filepath1', 'filepath2', source='XRS')   # doctest: +SKIP
-    >>> my_ts_list = ts.TimeSeries('/goesdirectory/', source='XRS')   # doctest: +SKIP
-    >>> my_ts_list = ts.TimeSeries(glob, source='XRS')   # doctest: +SKIP
-
-When manually specifying the source this functionality will only work with files from the same single source, generating a source specific child of the `~sunpy.timeseries.GenericTimeSeries` class such as the `~sunpy.timeseries.sources.goes.XRSTimeSeries` above.
+    >>> sunpy.data.sample.GOES_XRS_TIMESERIES  # doctest: +REMOTE_DATA
+    PosixPath('.../go1520110607.fits')
+    >>> my_timeseries = sunpy.timeseries.TimeSeries(sunpy.data.sample.GOES_XRS_TIMESERIES)  # doctest: +REMOTE_DATA
 
-Instead of creating a list of one TimeSeries object per file, you can create a single time series from multiple files using the keyword argument ``concatenate=True``:
+In many cases, sunpy will automatically detect the type of the file as well as the instrument associated with it.
+In this case, we have a FITS file containing an X-ray light curve as observed by the the XRS instrument on the GOES satellite.
 
-.. code-block:: python
-
-    >>> my_ts = ts.TimeSeries(sunpy.data.sample.GOES_XRS_TIMESERIES, sunpy.data.sample.GOES_XRS_TIMESERIES, source='XRS', concatenate=True)  # doctest: +REMOTE_DATA
+.. note::
 
-Again these must all be from the same source if the ``source`` keyword is explicitly specified.
-The `.GenericTimeSeries.concatenate` method can be used to make a single time series from multiple TimeSeries from different sources if they are already in the form of TimeSeries objects.
+    Time series data are stored in a variety of file types (e.g. FITS, csv, CDF), and so it is not always possible to detect the source.
```

Comment:
it could be good to mention here something along the lines of that you can create your custom series with a data source thats not supported, or open an issue requesting a new feature etc

Reply:
As with my above comment, I think this is not within scope. A user who has never used TimeSeries does not need to know about this at this point. If/when they do, we have sections of the docs that describe that process.